src/pybind/mgr/dashboard/tox.ini

```diff
@@ -105,6 +105,7 @@ deps =
     {[base-lint]deps}
 commands =
     flake8
+    flake8 --config=tox.ini /ceph/qa/tasks/mgr/dashboard
```

Comment:
This seems to only work in the ceph-dev-docker environment, maybe we can use a relative path?